bindata/network/multus/multus.yaml

```diff
@@ -121,6 +121,11 @@ data:
         "logToStderr": true,
         "logLevel": "verbose",
         "binDir": "{{ .CNIBinDir }}",
+        "perNodeCertificate": {
```

Comment:
Looks like currently multus hard-codes cert-duration to 10min: https://github.com/k8snetworkplumbingwg/multus-cni/pull/1159/files#diff-562bab31c287044ff5fd9a5ba21041103225e129ca4373634b389356bdce0070R101

 I think that the cert rotation shouldn't happen that often in a production environment.
In comparison, kubelet and ovn-kube ruse 24h.